frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/EditPageId.test.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { act, screen, waitFor } from '@testing-library/react';
+import { formLayoutSettingsMock, renderWithProviders } from '../../../testing/mocks';
+import { QueryKey } from 'app-shared/types/QueryKey';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+import userEvent from '@testing-library/user-event';
+import { textMock } from '../../../../../../testing/mocks/i18nMock';
+import { EditPageId } from './EditPageId';
+import type { ServicesContextProps } from 'app-shared/contexts/ServicesContext';
+
+// Test data
+const app = 'app';
+const org = 'org';
+const selectedLayout = 'layoutPageName';
+const layoutSetName = 'test-layout-set';
+
+describe('EditPageId', () => {
+  it('renders given page ID', () => {
+    renderEditPageId();
+    screen.getByRole('button', { name: textMock('ux_editor.id_identifier') });
+  });
+
+  it('calls updateFormLayoutName and textIdMutation with new page ID when changed', async () => {
+    const user = userEvent.setup();
+    const newPageName = 'myNewPageName';
+    const updateTextId = jest.fn();
+    const updateFormLayoutName = jest.fn();
+    const mockQueries: Partial<ServicesContextProps> = {
+      updateTextId,
+      updateFormLayoutName,
+    };
+    renderEditPageId(mockQueries);
+    const pageIdButton = screen.getByRole('button', { name: textMock('ux_editor.id_identifier') });
+    await act(() => user.click(pageIdButton));
+    const editPageId = screen.getByLabelText(
+      textMock('ux_editor.modal_properties_textResourceBindings_page_id'),
+    );
+    await act(() => user.clear(editPageId));
+    await act(() => user.type(editPageId, newPageName));
+    user.tab();
+    await waitFor(() => expect(updateFormLayoutName).toHaveBeenCalledTimes(1));
```

Comment:
Dette vil nok fungere, men hvis vi setter `await` foran `user.tab()` i stedet, blir det tydeligere hva vi venter på:
```suggestion
    await act(() => user.tab());
    expect(updateFormLayoutName).toHaveBeenCalledTimes(1);
```